src/Microsoft.Diagnostics.EventFlow.Inputs.Serilog/LoggerSinkConfigurationEventFlowExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Diagnostics.EventFlow;
+using Microsoft.Diagnostics.EventFlow.Inputs;
+using Serilog.Configuration;
+using Serilog.Debugging;
+using System;
+using System.Linq;
+
+namespace Serilog
+{
+    /// <summary>
+    /// Extends <see cref="LoggerSinkConfiguration"/>
+    /// </summary>
+    public static class LoggerSinkConfigurationEventFlowExtensions
+    {
+        /// <summary>
+        /// Publish Serilog events through a <see cref="DiagnosticPipeline"/>.
+        /// </summary>
+        /// <param name="loggerSinkConfiguration">The <c>WriteTo</c> object exposed by <see cref="LoggerConfiguration"/>.</param>
+        /// <param name="diagnosticPipeline">A configured <see cref="DiagnosticPipeline"/>. At least one Serilog input
+        /// must be configured for the pipeline.</param>
+        /// <returns>A <see cref="LoggerConfiguration"/> allowing configuration to continue.</returns>
+        /// <remarks>The method will select the first <see cref="SerilogInput"/> in the pipeline. If a specific input is desired instead,
+        /// this can be passed to <see cref="LoggerSinkConfiguration.Sink(Core.ILogEventSink, Events.LogEventLevel, Core.LoggingLevelSwitch)"/>
+        /// instead.</remarks>
+        public static LoggerConfiguration EventFlow(this LoggerSinkConfiguration loggerSinkConfiguration, DiagnosticPipeline diagnosticPipeline)
+        {
+            if (loggerSinkConfiguration == null) throw new ArgumentNullException(nameof(loggerSinkConfiguration));
```

Comment:
(minor) We already take a dependency on the Validation library elsewhere in EventFlow code, so the null checks can be simplified to Requires.NotNull(inputParameter, nameof(inputParameter)